framework/decode/vulkan_replay_dump_resources_common.cpp

```diff
@@ -108,7 +108,7 @@ static VkFormat ChooseDestinationImageFormat(VkFormat format)
 
     if (vkuFormatIsSRGB(format))
     {
-        dst_format = vkuFormatHasAlpha(format) ? VK_FORMAT_B8G8R8A8_SRGB : VK_FORMAT_B8G8R8_SRGB;
+        dst_format = VK_FORMAT_B8G8R8A8_SRGB;
```

Comment:
This value is used at line 453 and is then passed on to `dst_format` and through to `dumped_image.dumped_format`.  Does that mean that some images that have no alpha will end up dumped with 0 alpha?

Reply:
> Does that mean that some images that have no alpha will end up dumped with 0 alpha?

I don't think this can happen.
Images without alpha will be converted with `vkCmdBlitImage` into the R8G8B8A format that is chosen here and, if I understand this correctly, according to spec they will get an alpha value of 1

Reply:
The output json should list the image with its original format so this should be evident

Reply:
Excellent, thanks!